RFS/src/main/java/org/opensearch/migrations/bulkload/common/LuceneDocumentsReader.java

```diff
@@ -210,22 +318,19 @@ protected RfsLuceneDocument getDocument(IndexReader reader, int docSegId, boolea
                         break;
                 }
             }
-            if (id == null) {
-                log.atWarn().setMessage("Skipping document segment id {} from source {}, it does not have an referenceable id.")
-                    .addArgument(docSegId)
-                    .addArgument(indexDirectoryPath)
-                    .log();
+            if (openSearchDocId == null) {
+                log.atError().setMessage("Document with index {} does not have an id. Skipping")
```

Comment:
Without that index directory path there might not be context to know how to refine the source of the failure - this was just added because CX reported this condition that was unactionable.

Reply:
Unless there is something incorrect about the previous log messages, please keep these details in place.